test/integrations/destinations/drip/router/data.js

```diff
@@ -0,0 +1,246 @@
+const data = [
+  {
+    name: 'drip',
+    description: 'Test 0',
```

Comment:
We have to update the description with the appropriate one. This general description won't work. 
You can tag the person who has worked on drip to update that.